src/Components/Navbar.js

```diff
@@ -0,0 +1,28 @@
+import Link from 'next/link';
+import Icon from './UI/Icons/icons';
+import NavWrap from './UI/NavBar/NavWrap.styles';
+import NavigationButton from './UI/NavBar/NavigationButton.styles';
+import NavList from './UI/NavBar/NavList.styles';
+
+export default function Navbar() {
+	return (
+		<NavWrap>
+			<NavList>
+				<li>
+					<Link passHref href="/">
```

Comment:
Not sure if you need the `passHref`?

Reply:
Please have a look at my other comment. You might need this prop here.